src/components/table/header.rs

```diff
@@ -2,6 +2,24 @@ use super::column::TableColumn;
 use std::fmt::Debug;
 use yew::prelude::*;
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct TableHeaderSortBy<K>
+where
+    K: Clone + Eq + 'static,
+{
+    pub index: K,
+    pub asc: bool,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct TableHeaderContext<K>
+where
+    K: Clone + Eq + 'static,
+{
+    pub sort_by: Option<TableHeaderSortBy<K>>,
+    pub on_sort_by: Callback<Option<TableHeaderSortBy<K>>>,
```

Comment:
We do use `onsort` in the other case. Although `onsort_by` seems a bit weird, I would try to keep the pattern. Or use just `onsort`.

Reply:
variables were renamed